apps/web/components/connected/vehicles/PlateAvailableSearch.tsx

```diff
@@ -0,0 +1,162 @@
+import { useLazyQuery } from '@apollo/client'
+import { useState } from 'react'
+import {
+  AlertMessage,
+  AsyncSearchInput,
+  Box,
+  Text,
+} from '@island.is/island-ui/core'
+import {
+  ConnectedComponent,
+  Query,
+  QueryPlateAvailableArgs,
+} from '@island.is/web/graphql/schema'
+import { useNamespace } from '@island.is/web/hooks'
+import { PLATE_AVAILABLE_SEARCH_QUERY } from '@island.is/web/screens/queries/PublicVehicleSearch'
+
+import * as styles from './PlateAvailableSearch.css'
+
+const REPLACEMENT_KEY = '{{USER_INPUT}}'
```

Comment:
minor comment, is there potentially a more descriptive variable name than replacement key?

Reply:
I renamed it to be "PLATE_NUMBER_REPLACEMENT_KEY" which is a bit more descriptive